### PR TITLE
Addmetoqueue

### PR DESCRIPTION
**Pull request by TiredOfThisAll**

### CODE REVIEW

File: controller.py [hunk 1]
```diff
@@ -4,14 +4,17 @@
 
 NEW_QUEUE_COMMAND_RESPONSE_TEXT \
     = "Введите имя новой очереди в ответ на это сообщение"
+ADD_ME_TO_QUEUE_RESPONSE_TEXT \
```
Comment (levant47) [inline controller.py:7]: This constant is obsolete, we should remove it

File: controller.py [hunk 2]
```diff
@@ -137,6 +177,11 @@ def handle_show_queue_callback(
         try:
             queue_id = callback_query_data["queue_id"]
             queue_name = self.repository.get_queue_name_by_queue_id(queue_id)
+            if queue_name is None:
+                self.telegram_message_manager.send_message(
+                    callback_query["message"]["chat"]["id"],
+                    "Очереди с ID: " + queue_id + " не существует"
+                )
```
Comment (levant47) [inline controller.py:184]: There's a `return` statement missing here at the end of this `if` branch

File: main.py [hunk 3]
```diff
@@ -86,13 +90,28 @@
...
-    except Exception as error:
-        print(error)
+            except KeyboardInterrupt:
+                exit()
+            except HTTPError as http_error:
+                print("Encountered an HTTP error")
+                print("Stack trace:")
+                print(traceback.format_exc())
+                print("URL: " + http_error.url)
+                print("Response: " + http_error.file.read().decode("UTF-8") \
```
Comment (levant47) [inline main.py:112]: `./main.py:112:77: E502 the backslash is redundant between brackets`

File: bot_commands.json [hunk 4]
```diff
@@ -8,7 +8,11 @@
         "description": "create new queue"
     },
     {
-        "command": "/addmetoqueue",
-        "description": "Add yourself into the queue"
+      "command": "/addmetoqueue",
```
Comment (aznie) [inline bot_commands.json:11]: Inconsistent formatting, use 4 space instead of 2

Comment (levant47) [reply bot_commands.json:11]: You're welcome to fix this inconsistency in a separate pull request!

**Pull request merged**